app/main.py

```diff
@@ -28,15 +29,20 @@ def read_users_endpoint(db: Session = Depends(get_db)):
     return read_awards(db=db)
 
 
+@app.get("/read-onesite/", response_model=List[SchemaCreateOnesiteDB])
```

Comment:
Thank you for this! Please change `SchemaCreateOnesiteDB` to better fit the naming conventions set forth in `table_models.py`